docroot/themes/contrib/civictheme/webpack/webpack.common.js

```diff
@@ -156,6 +156,9 @@ module.exports = {
       '@organisms': path.resolve(__dirname, '../components/03-organisms'),
       '@templates': path.resolve(__dirname, '../components/04-templates'),
       '@pages': path.resolve(__dirname, '../components/05-pages'),
+      'twig': path.resolve(__dirname, '../node_modules/twig'),
+      'lorem-ipsum': path.resolve(__dirname, '../node_modules/lorem-ipsum'),
+      '@storybook': path.resolve(__dirname, '../node_modules/@storybook'),
```

Comment:
@AlexSkrypnyk 
@joshua-salsadigital 

I have added this to fix the resolver with the correct node_modules. This could be a temporary solution for now.